lib/input_sanitizer/version.rb

```diff
@@ -1,3 +1,3 @@
 module InputSanitizer
-  VERSION = "0.5.0"
+  VERSION = "0.5.1"
```

Comment:
I would go with 0.6.0. It's unlikely, but some clients may depend on current contract (using `nested` when `custom` would be better fit) - from what I see there is no validation if `:sanitizer` given to `nested` is a `InputSanitizer::V2::PayloadSanitizer` type.

Reply:
Fixed